wgs/workflows/cohort_qc/tasks.py

```diff
@@ -4,23 +4,75 @@
 from wgs.utils import helpers
 
 
-def merge_mafs(mafs, merged_maf, labels):
+
+def merge_cna_tables(amps, dels, labels, output, cohort):
+    cna = {d1: [a,d] for (d1, a), (d2, d) in zip(amps.items(), dels.items()) }
+    number=0
+    for s, files in cna.items():
+        
+        label = labels[(cohort, s)]
+        for f in files:
+        
+            data = pd.read_csv(f, usecols=["cn_type", "gene_name", "pass_filter"])
+
+            data=data[data.pass_filter == True]
+
+            data=data.rename(columns={"cn_type": "CN", "gene_name":"Gene"})
+
+            data["Sample_name"] = [label] * len(data)
+
+            data = data[["Gene", "Sample_name", "CN"]]
+
+            if number==0:
+                header=True
+            else:
+                header=False
+
+            if not data.empty:
+                number+=1
+                data.to_csv(output, index=False, mode='a', header=header, sep="\t")
+
+
+def classify_remixt(sample_label, remixt, gtf, output_dir, amps, dels, docker_image=None):
+
+    cmd = [
+        "classifycopynumber", gtf, output_dir, sample_label, amps, dels, "--remixt_h5_filename", remixt, "--plot", False
+    ]
+
+    pypeliner.commandline.execute(*cmd, docker_image=docker_image)
+
+
+
+def merge_mafs(mafs, merged_maf, write_header=True):
+    for m in mafs:
+
+        maf = pd.read_csv(m, sep="\t", chunksize=10e6)
+        for chunk in maf:
+
+            chunk.to_csv(merged_maf, sep="\t", index=False, header=write_header, mode='a')
+        #only write the first header
+        write_header=False   
+
+
+def merge_relable_mafs(mafs, merged_maf, labels, class_label=None, write_header=True):
 
     mafs = list(mafs.values())
     labels = list(labels.values())
 
     #only write the first header
-    write_header = True
-
+    write_header = write_header
```

Comment:
what??

Reply:
write_header is an argument

Reply:
write_header = write_header

is redundant